docs/gke/configs/cluster.jinja

```diff
@@ -286,9 +300,40 @@ TODO(jlewi): Do we need to serialize API activation
           members:
             - {{ 'serviceAccount:' + env['project_number'] + '@cloudservices.gserviceaccount.com' }}
 
+        {# servicemanagement.admin is needed by CloudEndpoints controller
```

Comment:
How do we manage deletion of IAM Role Bindings? When we delete the deployment, the service accounts go away but these IAM Role Bindings still lurk around.

Reply:
Good point. Created 
https://github.com/kubeflow/kubeflow/issues/910